ipaserver/install/ipa_restore.py

```diff
@@ -831,7 +832,7 @@ def cert_restore_prepare(self):
 
     def cert_restore(self):
         try:
-            certdb.update_ipa_nssdb()
+            update_ipa_nssdb()
```

Comment:
NOTE: This is probably the only exception of update_ipa_nssdb usage on server side contrary to what the commit message says but it seems fine to be used like this.